packages/interactivity/src/utils.ts

```diff
@@ -138,9 +163,9 @@ export function useWatch( callback ) {
  * This hook makes the element's scope available so functions like
  * `getElement()` and `getContext()` can be used inside the passed callback.
  *
- * @param {Function} callback The hook callback.
+ * @param callback The hook callback.
  */
-export function useInit( callback ) {
+export function useInit( callback: Function ) {
```

Comment:
If we an import for `type EffectCallback` from `preact`, we can use that where `_useEffect` is called:

```suggestion
export function useInit( callback: EffectCallback ) {
```